editReading.php

```diff
@@ -0,0 +1,23 @@
+<? 
+include('config.php'); 
+if (isset($_GET['device_id']) && isset($_GET['time']) ) { 
+$device_id = $_GET['device_id']; 
+$time = $_GET['time']; 
+if (isset($_POST['submitted'])) { 
+foreach($_POST AS $key => $value) { $_POST[$key] = mysql_real_escape_string($value); } 
+$sql = "UPDATE `reading` SET  `device_id` =  '{$_POST['device_id']}' ,  `time` =  '{$_POST['time']}' ,  `dimension0` =  '{$_POST['dimension0']}' ,  `dimension1` =  '{$_POST['dimension1']}'   WHERE  `device_id` = $device_id AND `time` = $time"; 
```

Comment:
There's something weird going on with `DATETIME` equality. The row doesn't actually get edited.